### PR TITLE
Test boot param fix

### PR DESCRIPTION
**Pull request by isaacdorfman**

### CODE REVIEW

**Review by jun-rh — CHANGES_REQUESTED** [review]
Fix the image building error in smoke test
Separate the boot param test fix to a different PR

File: Dockerfile [hunk 1]
```diff
@@ -40,16 +44,26 @@ ENV PATH=${PATH}:"/usr/local/go/bin":${GOPATH}/"bin"
...
 RUN git -C ${TNF_SRC_DIR} checkout ${GIT_CHECKOUT_TARGET}
 
+# Clone the partner source repository and checkout the target branch/tag/commit
+RUN git clone --no-single-branch --depth=1 ${TNF_PARTNER_SRC_URL} ${TNF_PARTNER_SRC_DIR}
+RUN git -C ${TNF_PARTNER_SRC_DIR} fetch origin ${GIT_PARTNER_CHECKOUT_TARGET}
+RUN git -C ${TNF_PARTNER_SRC_DIR} checkout ${GIT_PARTNER_CHECKOUT_TARGET}
+
 # Build TNF binary
 WORKDIR ${TNF_SRC_DIR}
 # TODO: RUN make install-tools
```
Comment (edcdavid) [inline Dockerfile:69]: Is call to "make local-test-infra" missing here?

File: local-test-infra.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+mkdir -p ./temp
+
+export res=$(oc get pod -A -l test-network-function.com/generic=fs_diff_master 2>&1)
+if [ "$res" == "No resources found" ]; then
+    cat $TNF_PARTNER_SRC_DIR/local-test-infra/fsdiff-pod.yaml | ./script/mo > ./temp/rendered-fsdiff-template.yaml
```
Comment (edcdavid) [inline local-test-infra.sh:5]: Why not run "make clean" and "make install" in the partner repo? Usually apply handles already running pods and the creation logic is already implemented in the partner repo. We could also use mo only in the partner repo.

File: Dockerfile (hunk 1 above)
Comment (isaacdorfman) [reply Dockerfile:69]: I just looked and apparently no need to.
The github action for tnf triggers the github action for the partner repo which runs "make install" which applies the yamls.

**Review by edcdavid — APPROVED** [review]

**Comment by pkliczewski** [comment]
@isaacdorfman please rebase

**Comment by jun-rh** [comment]
The original changes for adding fsdiff pod had been moved to the partner repo. Updating the PR title to reflect the changes left.

**Review by jun-rh — APPROVED** [review]

**Comment by isaacdorfman** [comment]
The boot params test originally run multiple "oc debug" sessions consecutively to get all relevant data.
This caused some problem with Openshift.
To mitigate them, I changed the test to get everything in one pipelined command on a single "oc debug" session

**Review by HamadiSE — APPROVED** [review]

**Pull request merged**